src/tests/integrationTests/ForceCalculationTest.cpp

```diff
@@ -80,6 +80,7 @@ void ForceCalculationTest::testForcePotentialCalculationF0() {
 	// -> potential U = -1 per particle pair.
 	ParticleContainer* container = initializeFromFile(ParticleContainerFactory::LinkedCell, "ForceCalculationTestF0.inp", 1.3);
 
+	ASSERT_EQUAL(4ul, container->getNumberOfParticles());
```

Comment:
It might be better to specify the type in `getNumberOfParticles()` cause this method has a bad default behavior depending on the particle container.

Reply:
done in 3fd77f0